module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -358,16 +358,16 @@ namespace module::input {
 
 
                         // This checks for an error on the platform and reports it
-                        if (input.platform_error_flags != RawSensors::Error::OK) {
+                        if (input.platform_error_flags != RawSensors::Error::EVERYTHING_OK) {
```

Comment:
Good god! Why?

Reply:
```
In file included from /home/nubots/NUbots/module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.hpp:25,
                 from /home/nubots/build/nuclear/roles/allmodules.cpp:14:
/home/nubots/build/nuclear/message/protobuf/message/platform/RawSensors.pb.h:1903:26: error: expected unqualified-id before numeric constant
 1903 |   static constexpr Error OK =
      |                          ^~
```
Something (outside of our codebase) has defined `OK`. I think it's ncurses, because [here](https://github.com/NUbots/NUbots/blob/main/module/behaviour/tools/ScriptTuner/src/ScriptTuner.cpp#L24) we undefine it. In keyboardwalk we [include it](https://github.com/NUbots/NUbots/blob/main/module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.hpp#L25) without undefining it though, and I think that's _probably_ the clash.

Reply:
find the offending header file and then `#undef OK` rather than changing our enum values as changing our enum values doesnt actually solve the problem

Reply:
sure